cmd/server/handler.go

```diff
@@ -174,10 +182,35 @@ func setup(d driver.Driver, cmd *cobra.Command) (admin *x.RouterAdmin, public *x
 	publicmw.Use(negronilogrus.NewMiddlewareFromLogger(d.Registry().Logger().(*logrus.Logger), fmt.Sprintf("hydra/public: %s", d.Configuration().IssuerURL().String())))
 	publicmw.Use(d.Registry().PrometheusManager())
 
-	metrics := metricsx.New(
-		cmd,
-		d.Registry().Logger(),
-		&metricsx.Options{
+	var (
+		bridges = []metricsx.Bridge{}
+		l       = d.Registry().Logger()
+		ctx     = context.Background()
+	)
+
+	optOut, err := cmd.Flags().GetBool("sqa-opt-out")
```

Comment:
Do we need all of this here or could we hide that in `metricsx` as it was previously?